std/range/package.d

```diff
@@ -908,8 +908,7 @@ if (Ranges.length > 0 &&
                 // @@@BUG@@@
```

Comment:
I tracked it down to this commit: https://github.com/D-Programming-Language/phobos/commit/69e5a68a7ef3274ebb0124df2546891f7dd32ddc

I would guess the problem is likely that front is overloaded, and you used to not be able to overload templates with normal functions. There isn't any more info in the commit message above, I'd say revert the "bug" and see if it works!